playbooks/group_vars/rabbitmq/public.yml

```diff
@@ -1,55 +0,0 @@
----
```

Comment:
Can you move this back to the public repo?

Reply:
Done 👍